schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -67,10 +67,16 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2015-04-08/Microsoft.DocumentDB.json#/resourceDefinitions/databaseAccounts_apis_keyspaces_tables" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-04-08/Microsoft.DocumentDB.json#/resourceDefinitions/databaseAccounts_apis_tables" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-06-01/Microsoft.KeyVault.json#/resourceDefinitions/vaults" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-06-01/Microsoft.KeyVault.json#/resourceDefinitions/secrets" },
```

Comment:
IIRC, this resource is useful in ARM template, so ARM wants to keep it. @anthony-c-martin could you confirm?

Reply:
Yup, that's correct.